src/crimson/osd/pg.cc

```diff
@@ -145,21 +145,24 @@ void PG::publish_stats_to_osd()
 {
   if (!is_primary())
     return;
+  const bool is_stats_valid = pg_stats.has_value();
+  if (auto new_pg_stats = peering_state.prepare_stats_for_publish(
+	is_stats_valid,
+	pg_stats.value_or(pg_stat_t{}),
+	object_stat_collection_t());
+      new_pg_stats.has_value()) {
```

Comment:
OK, looks the indention is correct but GitHub introduces some visual mess (tab has effectively 6 spaces).

Reply:
i add a commit to use spaces instead of tab. this tab versus spaces thing has been a plain when reviewing the change using github web interface.